client/layout/index.jsx

```diff
@@ -409,7 +409,7 @@ export default withCurrentRoute(
 			'plugins',
 			'comments',
 		].includes( sectionName );
-		const sidebarIsHidden = ! secondary || isWcMobileApp();
+		const sidebarIsHidden = ! secondary || isWcMobileApp() || currentQuery?.[ 'is-launchpad-step' ];
```

Comment:
Any reason not to use `getQueryArgs()` here?

Reply:
> I'm just doing a drive-by review in case that helps.

Thanks for having a look @tyxla 👍

Reply:
In this context `currentQuery` is already populated with the query args object, no need to import something